Sources/TokamakDemo/ListDemo.swift

```diff
@@ -44,6 +44,6 @@ public struct ListDemo: View {
           Text("Item: \($0 + 1)")
         }
       }
-    }
+    }.frame(width: 400, height: 300)
```

Comment:
Not sure if this `frame` change is related to this PR?

Reply:
I see, it collapses the list 👍

Reply:
I reverted it anyway

Reply:
I guess it won't be needed as much when it's more organized after the navigation view PR is merged...